### PR TITLE
fix(rollback): Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by abhiomkar**
PiperOrigin-RevId: 283418957

This is initial export from internal to GitHub (mandatory) to setup Copybara for 2-way sync.

### CODE REVIEW

**Comment by patrickrodee** [comment]
Hey, I'm not quite following what's going on here. Is this to get GitHub and google3 into the same state?

**Comment by abhiomkar** [comment]
> Hey, I'm not quite following what's going on here. Is this to get GitHub and google3 into the same state?

Hey -- Yes, we need to perform initial export to keep internal and github repos on same state. This is mandatory to initialize repo for Copybara. After this commit Copybara would auto generate CL per PR, or PR per CL. See http://go/copybara-piper-sot#first_run for more info.

File: packages/mdc-chips/mdc-chips.scss [hunk 1]
```diff
@@ -22,4 +22,4 @@
 //
 
 @import "./chip/mdc-chip";
-@import "./chip-set/mdc-chip-set";
+@import "./chipset/mdc-chip-set";
```
Comment (patrickrodee) [inline packages/mdc-chips/mdc-chips.scss]: This needs to be updated.

File: packages/mdc-menu/package.json [hunk 2]
```diff
@@ -20,7 +20,6 @@
   "dependencies": {
     "@material/base": "^4.0.0",
     "@material/dom": "^4.0.0",
-    "@material/elevation": "^4.0.0",
```
Comment (patrickrodee) [inline packages/mdc-menu/package.json]: The elevation mixins are still used on line 37 of mdc-menu/_mixins.scss.

**Comment by abhiomkar** [comment]
Changes reverted in other PRs. Closing this.

**Pull request closed without merging**